.github/workflows/release.yml

```diff
@@ -179,13 +178,13 @@ jobs:
           SLACK_WEBHOOK_TYPE: "INCOMING_WEBHOOK"
 
   deploy-sandbox:
-    if: startsWith(github.event.release.tag_name, 'v2.')
+    if: "${{ ! github.event.release.draft && ! github.event.release.prerelease }}"
```

Comment:
The prerelease workflow was deploying next.demo. I think that was inaccurate though since `next.demo` always pushes from the `next` branch and a prerelease doesn't always necessarily mean to deploy the `next` branch?

Reply:
Does this workflow run on v1.6 releases? I think it may use the workflow from the tagged commit so it's ok to remove the `if: startsWith(github.event.release.tag_name, 'v2.')`?

Reply:
That's my belief/expectation yes, that `ltm-1.6` will use its own version of the workflow if we ever release another 1.6 version. Hopefully that is correct.